src/full/Agda/Main.hs

```diff
@@ -44,14 +42,10 @@ import qualified Agda.Utils.Benchmark as UtilsBench
 import Agda.Utils.Except ( MonadError(catchError, throwError) )
 import Agda.Utils.Impossible
 
-builtinBackends :: [Backend]
-builtinBackends = [ ghcBackend, jsBackend ]
-
 -- | The main function
-runAgda :: [Backend] -> IO ()
+runAgda', runAgda :: [Backend] -> IO ()
```

Comment:
When types are short (like in this case), I'd prefer two separate type signatures (like it was before).  I think code is easier to read when it is uniform so that the type signature always just comes before the definition.